target/snitch_cluster/sw/toolchain.mk

```diff
@@ -18,18 +18,19 @@ DEBUG ?= OFF # ON to turn on debugging symbols
 # Compiler toolchain
 ifeq ($(SELECT_TOOLCHAIN), llvm-snitch)
 # specialized version does not use a version specifier in the binary
-LLVM_BINROOT    = /tools/riscv-llvm/bin
+LLVM_BINROOT    = /tools/riscv-llvm/bin/
 LLVM_VERSION    = 
+RISCV_LD        ?= $(LLVM_BINROOT)ld.lld$(LLVM_VERSION)
 else
-LLVM_BINROOT    = /usr/bin
+LLVM_BINROOT    =
 LLVM_VERSION    =
+RISCV_LD        ?= $(LLVM_BINROOT)lld$(LLVM_VERSION)
```

Comment:
rather strangely, the newer version seems to be completely okay with invoking it as just lld